src/python/pants/backend/google_cloud_function/python/rules_test.py

```diff
@@ -103,7 +103,7 @@ def complete_platform(rule_runner: RuleRunner) -> bytes:
         {
             "pex_exe/BUILD": dedent(
                 """\
-                python_requirement(name="req", requirements=["pex==2.1.66"])
+                python_requirement(name="req", requirements=["pex==2.1.99"])
```

Comment:
The tests were failing for me since I'm defaulting to Python 3.11 right now. Pex 2.1.66 does not support 3.11.